core/tests/org.fusesource.ide.camel.model.service.core.tests.integration/src/main/java/org/fusesource/ide/camel/model/service/core/tests/integration/core/model/DataFormatsInCamelModelElementIT.java

```diff
@@ -78,4 +79,12 @@ public void testDataFormatsInUnmarshal_notRemovedWhenUpdatingRefParameter() thro
 		assertThat(((AbstractCamelModelElement) unmarshallElement.getParameter("dataFormatType")).getNodeTypeId()).isEqualTo("json");
 	}
 
+	@Test
+	public void testCanFindTypes() throws CoreException, IOException {
+		String[] typesToFind = {"csv", "json", "string"};
```

Comment:
I could be wrong but shouldn't it be more like 

`String[] typesToFind = {"bindy", "json"};`

And shouldn't we check for result set size > 1 ?

Reply:
http://joel-costigliola.github.io/assertj/core/api/org/assertj/core/api/AbstractIterableAssert.html#isNotEmpty()